src/plays/Selection-Sort-Visualizer/select.css

```diff
@@ -0,0 +1,85 @@
+body {
+  background-color: papayawhip;
+}
```

Comment:
please avoid targeting directly otherwise it will change the whole website's css